site-infrastructure.yml

```diff
@@ -252,6 +252,20 @@
     - deps-hbase
     - deps-spark
 
+# --------------------------------- Oozie ------------------------------------
+- name: Oozie Install
+  hosts: oozie-node
+  roles: [ oozie ]
+  vars:
+    - spark_master_enabled: false
+    - spark_worker_enabled: false
+    - dfs_namenode_host: "{{ groups['hadoop-namenode-node'][0] }}"
+    - yarn_resourcemanager_hostname: "{{ groups['hadoop-namenode-node'][0] }}"
+    - yarn_historyserver_hostname: "{{ groups['hadoop-namenode-node'][0] }}"
+    - zookeeper_host_list: "{{ groups['zookeeper-nodes'] }}"
```

Comment:
@madisonb I doubt they are needed anymore. They were added so the (now removed) dependencies would complete successfully